script/c79656239.lua

```diff
@@ -69,7 +69,7 @@ function s.descon(e,tp,eg,ep,ev,re,r,rp)
 	return ep==tp
 end
 function s.destg(e,tp,eg,ep,ev,re,r,rp,chk,chkc)
-	if chkc then return true end
+	if chkc then return chkc:IsLocation(LOCATION_ONFIELD) and chkc:IsControler(1-tp) end
```

Comment:
This change is incorrect, the effect is mandatory much like the Main Deck Marjoram correctly changed in this PR.

Reply:
`chkc` is for target redirecting check, so it is necessary even for mandatory effect

Reply:
Yes, you're right. Sorry I misread, I thought it was `chk`.